soh/soh/GameMenuBar.cpp

```diff
@@ -856,7 +856,8 @@ namespace GameMenuBar {
                     const char* bunnyHoodOptions[3] = { "Disabled", "Faster Run & Longer Jump", "Faster Run"};
                     UIWidgets::EnhancementCombobox("gMMBunnyHood", bunnyHoodOptions, 3, 0);
                     UIWidgets::Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask. The longer jump option is not accounted for in randomizer logic.");
-                    UIWidgets::PaddedEnhancementCheckbox("Mask Select in Inventory", "gMaskSelect", true, false);
+                    UIWidgets::PaddedEnhancementCheckbox("Mask Select in Inventory", "gMaskSelect", true, false, gSaveContext.n64ddFlag, 
+                        "This setting is always enabled on Randomizer saves");
```

Comment:
could you elaborate on this decision a little? i know this wasn't the case before, is mask select expected in logic now or something?

Reply:
@briaguya-ai So this logically shouldn't cause any differences for any rando, the player will only be able to access items they _should_ have access to. This just gives us more flexibility going forward to supporting getting the child trade sequence items in a different order than intended.

Starting with the bunny mask is one of the situations that is now handled gracefully. Previously to start with bunny hood you had to skip child zelda, start with the kak gate open, and skip the mask trade quest. Now you can start with the bunny mask without any of those options set, and everything should work fine, allowing you to swap between items when needed.

Reply:
i'm just curious as to why it's "always have mask select enabled when rando is enabled" thing instead of tied to the bunny hood in some way